jenkins/scripts/integration_test.sh

```diff
@@ -57,6 +60,22 @@ EXT_PORT_ID="$(openstack port create -f json \
   --fixed-ip subnet="$(get_subnet_name "${CI_EXT_NET}")" \
   "${TEST_EXECUTER_PORT_NAME}" | jq -r '.id')"
 
+# Get the base volume
+BASE_VOLUME_NAME_ID="$(openstack volume show -f json \
+  "${BASE_VOLUME_NAME}" | jq -r '.id')"
+
+# Create copy of base volume
+TEST_EXECUTER_VOLUME_ID="$(openstack volume create -f json \
+  --source "${BASE_VOLUME_NAME_ID}" \
+  "${TEST_EXECUTER_VOLUME_NAME}" | jq -r '.id')"
+
+# Create new executer vm from copy of the test executer volume
+openstack server create \
+  --volume "${TEST_EXECUTER_VOLUME_ID}" \
+  --flavor "${TEST_EXECUTER_FLAVOR}" \
+  --port "${EXT_PORT_ID}" \
+  "${TEST_EXECUTER_VM_NAME}"
+
 # Create new executer vm
 openstack server create -f json \
```

Comment:
You don't need these lines, since you create a VM in L73.

Reply:
Yeah agree, but did not we agree to do a cleanup of additional code after these changes are in place, in order to have a backwards compatibility?

Reply:
What I meant previously is that this patch should go after we start building volumes with the image building CI. Once volumes started to be created in CityCloud by the image building CI,  we trigger the integration CI on this PR to check if everything goes like clockwork.

Reply:
Okay, removed.